### PR TITLE
[web] Fixed index page.

### PR DESCRIPTION
**Pull request by bilke**
Also created the URL [benchmarks.opengeosys.org](https://benchmarks.opengeosys.org).

Checked reason why visualization of [const viscosity benchmark](https://dev.opengeosys.org/docs/benchmarks/hydro-thermal/constant-viscosity/) does not show up: it is a large benchmark and visualization are build just for regular benchmarks. I suggest to fix [remaining failing large benchmarks](https://jenkins.opengeosys.org/job/OGS-6/job/gcc-tests-large/191/execution/node/22/log/) and the run the large benchmarks on the regular ufz/master-job thus creating all visualizations.

### CODE REVIEW

File: web/content/docs/benchmarks/thermo-mechanics/thermomechanics.md [hunk 1]
```diff
@@ -31,9 +31,6 @@ $$
...
 $$
 
-The relative error between the numerical simulation and the analytical solution is 9.2 $\cdot$ 10\uexp{-13}.
-
-Kolditz, Olaf, Uwe-Jens G�rke, Hua Shao, and Wenqing Wang, eds. Thermo-hydro-mechanical-chemical processes in porous media: benchmarks and examples. Vol. 86. Springer Science & Business Media, 2012.
-
-
+The relative error between the numerical simulation and the analytical solution is $9.2 \cdot 10^{-13}$.
 
+Kolditz, Olaf, Uwe-Jens Görke, Hua Shao, and Wenqing Wang, eds. Thermo-hydro-mechanical-chemical processes in porous media: benchmarks and examples. Vol. 86. Springer Science & Business Media, 2012.
```
Comment (TomFischer) [inline web/content/docs/benchmarks/thermo-mechanics/thermomechanics.md]: Name schema in citation is inconsistent: Either: 'first name last name, ...' or 'last name first name'.

Comment (bilke) [inline web/content/docs/benchmarks/thermo-mechanics/thermomechanics.md]: Hmm, just fixed an umlaut...

**Comment by endJunction** [comment]
@bilke When web build is green :fast_forward: 
[There](https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-1883/3/execution/node/119/log) is an error building the web for now...

**Comment by bilke** [comment]
@endJunction I have bibtex references almost implemented and will add this functionality right here..

**Pull request merged**